KoPrivateGPT/benchmark/retrieval/retrieval_evaluation_master.py

```diff
@@ -0,0 +1,50 @@
+from base_retrieval_evaluation_factory import RecallFactory, RRFactory, PrecisionFactory, NDCGFactory, DCGFactory, HoleFactory, TopKAccuracyFactory, IDCGFactory, IndDCGFactory, IndIDCGFactory, APFactory, CGFactory
+from strategyQA import strategyQA
+
+
+from typing import List, Dict
+
+import json
+import click
+
+
+def retrieval_evaluation_master(qrels: Dict[str, Dict[str, int]],
```

Comment:
change function name and file name to another one. This is not "master". Just StrategyQA implementation of evaluation metrics

Reply:
...! gotcha